content/documentation/whitepaper.md

```diff
@@ -312,26 +309,28 @@ That is, git prepends the blob with the string **"commit"**, decimal-encoded siz
 This variance leads to the same content named and referenced in different formats across Merkle DAG-based
 systems.
 
-There is value in making Merkle links interoperatable across technologies. As an example, you would be able to reference a blockchain transaction in your git commit. It open even more possibilities with Ethereum dapps where you can link to the content stored elsewhere (be that IPFS or other cryptocurrencies) in your smart contracts.
+There is value in making Merkle links interoperable across technologies. As an example, you would be able to reference a blockchain transaction in your git commit. It open even more possibilities with Ethereum dapps where you can link to the content stored elsewhere (be that IPFS or other cryptocurrencies) in your smart contracts.
 
-To enable Merkle Link interoperability, Protocol Labs proposed IPLD, a universal content identifier format that supports multiple cryptohash formats used across systems. Adopters of IPLD are able to . Yet IPLD cannot be adopted retroactively to existing Merkle DAG blocks since changing the hash serialization format used in Merkle link changes the hash of the block itself which leads to breaking any existing Merkle links that are used to refer to this block itself.
+To enable Merkle Link interoperability, Protocol Labs proposed IPLD, a universal content identifier format that supports multiple cryptohash formats used across systems. Adopters of IPLD are able to. Yet IPLD cannot be adopted retroactively to existing Merkle DAG blocks since changing the hash serialization format used in Merkle link changes the hash of the block itself which leads to breaking any existing Merkle links that are used to refer to this block itself.
```

Comment:
Fyi, I don't think the sentence `Adopters of IPLD are able to.` makes sense in this context.